includes/class-form.php

```diff
@@ -147,4 +156,227 @@ public function get_subs_fallback_cost() {
+
+                } elseif ( !$pay_per_post && !$current_user->subscription()->has_post_count( $form_settings['post_type'] ) ) {
+
+                    $user_can_post = 'no';
+                    $info = sprintf( __( 'Payment type not selected for this form. Please contact admin.', 'wp-user-frontend' ));
+
+                } else {
+                    $user_can_post = 'no';
+                    if ( !is_user_logged_in() ) {
+                        $info = $form_settings['message_restrict'];
+                    } else {
+                        $info = sprintf( __( 'Payment type not selected for this form. Please contact admin.', 'wp-user-frontend' ));
+                    }
+
+                }
+            }
+        } else {
+            if ( isset( $form_settings['guest_post'] ) && $form_settings['guest_post'] == 'true' && ! is_user_logged_in() ) {
+                $user_can_post = 'yes';
+            }
+        }
+
+        return array($user_can_post,$info);
+    }
+
+
+    /**
+     * prepare_entries add by rokib
+     *
+     * @return array
+     */
+    public function prepare_entries() {
+
+        $fields       = wpuf()->fields->get_fields();
+        
+        $form_fields  = $this->get_fields();
+
+        $entry_fields = array();
+
+        $ignore_list  = apply_filters('wpuf_entry_ignore_list', array('recaptcha'));
+
+        foreach ($form_fields as $field) {
+
+            if ( in_array( $field['template'], $ignore_list ) ) {
+                continue;
+            }
+
+            if ( ! array_key_exists( $field['template'], $fields ) ) {
+                continue;
+            }
+
+            $field_class = $fields[ $field['template'] ];
+
+            $entry_fields[ $field['name'] ] = $field_class->prepare_entry( $field );
+        }
+
+        return apply_filters( 'wpuf_prepare_entries', $entry_fields );
+    }
+
+    /**
+     * Get all form fields of this form
+     *
+     * @return array
+    */
+    
+    public function get_fields() {
+
+        // return if already fetched
+        if ( $this->form_fields ) {
+            return $this->form_fields;
+        }
+
+        $fields = get_children(array(
+            'post_parent' => $this->id,
+            'post_status' => 'publish',
+            'post_type'   => 'wpuf_input',
+            'numberposts' => '-1',
+            'orderby'     => 'menu_order',
+            'order'       => 'ASC',
+        ));
+
+        $form_fields = array();
+
+        foreach ( $fields as $key => $content ) {
+
+            $field = maybe_unserialize( $content->post_content );
+
+            if ( empty( $field['template']  ) ) {
+                continue;
+            }
+
+
+            $field['id'] = $content->ID;
+
+            // Add inline property for radio and checkbox fields
+            $inline_supported_fields = apply_filters( 'inline_supported_fields_list', array( 'radio_field', 'checkbox_field' ) );

```

Comment:
Why the filters don't have a prefix. Is it new or old code? We should have a prefix. e.g. `wpuf_inline_....`